### PR TITLE
Search mode methods

### PR DESCRIPTION
**Pull request by kmathew-pdftron**

### CODE REVIEW

**Comment by terryyoon-pdftron** [comment]
@sgong-pdftron @JamesTron

File: android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java [hunk 1]
```diff
@@ -2566,7 +2609,8 @@ private static void groupAnnotations(MethodCall call, MethodChannel.Result resul
                 JSONObject annotationJson = new JSONObject(annotation);
                 String primaryAnnotationId = annotationJson.getString(KEY_ANNOTATION_ID);
                 int primaryAnnotPageNum = annotationJson.getInt(KEY_PAGE_NUMBER);
-                Annot primaryAnnotation = ViewerUtils.getAnnotById(pdfViewCtrl, primaryAnnotationId, primaryAnnotPageNum);
+                Annot primaryAnnotation = ViewerUtils.getAnnotById(pdfViewCtrl, primaryAnnotationId,
```
Comment (sgong-pdftron) [inline android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java:2612]: there are many strange formatting changes in this file, not sure what's happening

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java (hunk 1 above)
Comment (terryyoon-pdftron) [reply android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java:2612]: maybe it was done by a linter when Kevin was making changes to the file, should i try and revert them all?

Comment (sgong-pdftron) [reply android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java:2612]: try auto-reformat see if that fixes it, otherwise yes you will need to revert and do proper formatting using the formatter

**Review by sgong-pdftron — APPROVED** [review]

**Review by JamieDass — CHANGES_REQUESTED** [review]
Could we merge master in and resolve conflicts and I will re-review.

File: ios/Classes/PdftronFlutterPlugin.h [hunk 2]
```diff
@@ -475,26 +482,26 @@ typedef enum
...
+- (void)documentController:(PTDocumentController *)documentController formFieldValueChanged:(NSString *)fieldString;
+- (void)documentController:(PTDocumentController *)docVC behaviorActivated:(NSString *)behaviorString;
+- (void)documentController:(PTDocumentController *)docVC leadingNavButtonClicked:(nullable NSString *)nav;
+- (void)documentController:(PTDocumentController *)docVC longPressMenuPressed:(NSString *)longPressMenuPressedString;
+- (void)documentController:(PTDocumentController *)docVC annotationMenuPressed:(NSString *)annotationMenuPressedString;
+- (void)documentController:(PTDocumentController *)docVC leadingNavButtonClicked:(nullable NSString *)nav;
+- (void)documentController:(PTDocumentController *)docVC pageChanged:(NSString *)pageNumbersString;
+- (void)documentController:(PTDocumentController *)docVC zoomChanged:(NSNumber *)zoom;
+- (void)documentController:(PTDocumentController *)docVC pageMoved:(NSString *)pageNumbersString;
+- (void)documentController:(PTDocumentController *)docVC scrollChanged:(NSString*)scrollString;
```
Comment (JamieDass) [inline ios/Classes/PdftronFlutterPlugin.h:500]: I don't really understand these (and similar) changes. @terryyoon-pdftron any ideas? I guess this is also from before you took over.

File: doc/api/API.md [hunk 3]
```diff
@@ -92,16 +97,19 @@ PdftronFlutter.setRequestedOrientation(0);
 ```
 
 ## Viewer Functions
+
```
Comment (JamieDass) [inline doc/api/API.md:100]: Why are there so many formatting changes to the API file? Seems like it should be a separate PR.
I think this is from before you took over, @terryyoon-pdftron. I guess it's ok.

**Review by JamieDass — APPROVED** [review]
Approved but there are a number of seemingly unrelated formatting changes that shouldn't really be part of this PR as far as I can tell.
